satpy/writers/__init__.py

```diff
@@ -781,6 +781,10 @@ def save_dataset(self, dataset, filename=None, fill_value=None,
                             If `False` return either a :doc:`dask:delayed`
                             object or tuple of (source, target). See the
                             return values below for more information.
+            units (str or None): If not None, will convert the dataset to
+                                    the given unit before saving using
+                                    pint-xarray. Default is not to do any
```

Comment:
```suggestion
                                    the given unit using pint-xarray before
                                    saving. Default is not to do any
```

Otherwise it sounds like it is using pint-xarray for saving.

Reply:
Indeed, thanks